test/utils.ts

```diff
@@ -104,6 +108,8 @@ export const PRIZE_TRACKING_TICKET_PUBKEY = new PublicKey(
   '78qz3gehg9YqktdaYt6o99DSUPFQ41tLMACHpnFjdYdS',
 );
 
```

Comment:
Given this runs against local validator now why do we still need to match the feepayer in devnet?
You could just generate a random keypair each time ... it shouldn't matter as your code + tests should work no matter what that key is.

Reply:
I prefer to leave it how it is for now, other way I have to refactor many tests, which I'm going to do eventually step by step